src/btree/bt_page.c

```diff
@@ -312,6 +312,8 @@ __wt_page_inmem_prepare(WT_SESSION_IMPL *session, WT_REF *ref)
         }
     }
 
+    /* Mark the page clean after re-instantiating prepared updates. */
```

Comment:
Can you expand the comment to describe that we have written everything to disk so we can mark it clean to avoid reconcile the page again?

Reply:
Added a comment in 2f881514b6edf867d95c81d34619784092a3a1a2